### PR TITLE
Build 7.7 release candidate

### PR DESCRIPTION
**Pull request by peastman**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by peastman** [comment]
@conda-forge-admin, please rerender

**Comment by peastman** [comment]
@conda-forge/core what does the above message mean?

**Comment by beckermr** [comment]
See the error here: https://github.com/conda-forge/openmm-feedstock/runs/4475636163?check_suite_focus=true

**Comment by peastman** [comment]
Does that mean conda-forge has dropped support for CUDA 9.2 through 10.2?

**Comment by peastman** [comment]
@conda-forge-admin, please rerender

**Comment by peastman** [comment]
I tried removing the older CUDA versions, but it still gives the same message.  Any suggestions?

**Comment by peastman** [comment]
@conda-forge-admin, please rerender

**Comment by raimis** [comment]
@peastman is any reason why it is not being built for the latest CUDA versions? `cudatoolkit` 11.5 (https://anaconda.org/conda-forge/cudatoolkit) is already available.

**Comment by jaimergp** [comment]
We are using CUDA's forward compatibility. Packages built with 11.2 should work just fine for >=11.2, so there's no need to add more jobs to the (already saturated) matrix.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -26,9 +26,6 @@ cuda_compiler:  # [linux64 or win or (linux and ppc64le)]
 cuda_compiler_version:
```
Comment (jaimergp) [inline recipe/conda_build_config.yaml:26]: I think we can clear this file now.

Everything under `# When adding or removing cuda versions, make sure that the following entries are "zipped";` can be removed so we go back to conda-forge's default config after a rerender.

Comment (jaimergp) [reply recipe/conda_build_config.yaml:26]: We might need to keep the ppc64le parts though.

Comment (peastman) [reply recipe/conda_build_config.yaml:26]: Can you tell me exactly what I need to change?  I don't completely understand.

**Comment by peastman** [comment]
@conda-forge-admin, please rerender

**Comment by jaimergp** [comment]
Hm, looks like the PowerPC builds are gone. We might need to add the relevant entries back in the `conda_build_config.yaml`.

**Comment by peastman** [comment]
Can you tell me what to do?  I don't understand how the mechanism works, and there are a lot of lines with lots of different combinations of specifications.

**Comment by jaimergp** [comment]
I think we need this migrator https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2001 but it's not merged yet. I might have time by the end of the week to try a couple things. You can merge this RC for now and if anything we can open a new PR for ppc when I get to it.

**Comment by peastman** [comment]
Ok, thanks!

**Pull request merged**